aibolit/patterns/var_siblings/var_siblings.py

```diff
@@ -1,10 +1,20 @@
+import javalang
+import re
+from aibolit.utils.ast import AST
+
 
 class VarSiblings:
     def __init__(self):
         pass
 
```

Comment:
1) Remove P27 pattern from config.py
2) Check tests (see `CI_pipeline` log), they take too much time. Seems smth is wrong with your code.